### PR TITLE
Add readers and writers instances for cats-collections

### PR DESCRIPTION
**Pull request by Roman-Statsura**
Add readers and writers instances for cats-collections

### CODE REVIEW

**Comment by Roman-Statsura** [comment]
I have problems with some collections, such as Range: how does look like Range(1,5) in Json? I have two option: [1,2,3,4,5] and [1, 5]. Which option is correct? How to read Range from Json? It might not be worth implementing instances for this and anyone can do it themselves?

File: build.sbt [hunk 1]
```diff
@@ -9,7 +9,7 @@ lazy val scala3 = "3.3.0"
 ThisBuild / scalaVersion := scala3
 
 lazy val commonSettings = Seq(
-  version := "0.28.4",
```
Comment (dos65) [inline build.sbt:12]: Could you rebase commit on a master branch?
There is no more need to update version by hand

Comment (Roman-Statsura) [reply build.sbt]: ok I did it

**Comment by dos65** [comment]
@Roman-Statsura Sorry for delay! 
I just realized that it should be better to inroduce an additional model `tethys-cats-collection` rather that extendening existing one with `typelevel/cats-collections`.  Could you do that?

**Pull request closed without merging**